submissions/evgeniy24/html-css-popup-demo/styles/style.scss

```diff
@@ -0,0 +1,379 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Segoe UI", sans-serif;
+  font-size: 13px;
+  color: #686bb1;
+}
+
+body,
+ul,
+ol,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+hr,
+p,
+figure,
+nav,
+button {
+  outline: none;
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/*main menu*/
+.navigation {
+  padding: 10px 10px 0 0;
+}
+
+.navigation__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-box-pack: end;
+  -ms-flex-pack: end;
+  justify-content: flex-end;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.list__item {
```

Comment:
One element's focus color is different from the rest.

Reply:
What exactly? Sorry. Cant understand what do you mean!

Reply:
Another color and border-radius
![1](https://user-images.githubusercontent.com/26724390/97472711-ccdd4d00-1952-11eb-8bc9-8ff914deabc9.png)
![2](https://user-images.githubusercontent.com/26724390/97472716-cea71080-1952-11eb-82c7-647be16d0419.png)

Reply:
In the current task, the color difference is acceptable.